client/src/components/Button/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import classes from './style.css'
```

Comment:
add a new line after the import the React

Reply:
Done